assets/src/types.ts

```diff
@@ -2,17 +2,32 @@ export type User = {
   id: number;
   email: string;
   display_name?: string;
-  full_name?: string;
+  name?: string;
```

Comment:
why this change? I'm not sure this is correct 🤔

Reply:
![image](https://user-images.githubusercontent.com/25644166/96001490-b664cb00-0e55-11eb-87dc-8a1f25712596.png)

The BE API returns response having Customer object having property as `name` (not `full_name`), so to maintain uniformity (and to replace `any` with `Customer`), I replaced the variable name

Reply:
ah -- `name` is for the `Customer` type, but `full_name` is for the `User` type

Reply:
https://github.com/papercups-io/papercups/blob/4e9c4580faa7641ccc3adb338456cbcfeca0ab17/assets/src/components/conversations/ConversationHeader.tsx#L198
In this line `user.name` was used, that's why I must have changed the property. I think I should keep the property as `full_name` and change the corresponding line

Reply:
ah yes, that line is incorrect 👍

Reply:
Can you review again?